Tests/test_GenBank.py

```diff
@@ -22,6 +22,7 @@
 from Bio import BiopythonParserWarning
 
 from Bio import SeqIO
+from Bio.SeqFeature import SeqFeature, FeatureLocation
```

Comment:
These imports were so I could basically mimic the original issue's example script for reproducing the error. At first I tried making a new `.gb` file but it started to seem like overkill.